### PR TITLE
Remove policies.

### PR DESCRIPTION
**Pull request by knapii-developments**
Due to correctness concerns, policies that needed exception per-app basis have been removed. A README is included at the deployment level so that policies can be further explained. 

Successful linter build: https://github.com/broadinstitute/terra-helmfile/pull/1423/checks?check_run_id=3019852576

### CODE REVIEW

File: actions/terra-k8s-linter/policy/deployments/general_policies.rego [hunk 1]
```diff
@@ -24,15 +24,8 @@ deny_required_deployment_labels[msg] {
...
-  input.metadata.annotations["bio.terra.linter/replicas_exception"] != "enabled"
-  input.spec.replicas < min_required_replicas
-  msg := sprintf("Must have at least 3 replicas. %s has %d replicas.", [name, input.spec.replicas])
-}
-
 deny_revision_history[msg] {
   input.kind == "Deployment"
   input.spec.revisionHistoryLimit != revision_history_limit
-  msg := sprintf("%s should set revisionHistoryLimit to 0. Currently is: %b", [name, input.spec.revisionHistoryLimit])
+  msg = sprintf("%s should set revisionHistoryLimit to 0. Currently is: %d", [name, input.spec.revisionHistoryLimit])
```
Comment (jack-r-warren) [inline actions/terra-k8s-linter/policy/deployments/general_policies.rego]: Smallest of nits: maybe we could use `revision_history_limit` here instead of hardcoding 0

**Review by jack-r-warren — DISMISSED** [review]
LGTM!

File: actions/terra-k8s-linter/policy/deployments/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+### Deployment Linting policies
+
+#### General policies
+
+Policies can be found in [deny.rego](https://github.com/DataBiosphere/github-actions/blob/master/actions/terra-k8s-linter/policy/deployments/deny.rego). These are policies that can't don't fall in s particular group and are general in nature.
+* Deployments should set revisionHistoryLimit to 0
+    * The only reason to keep old revisions around is to use kubectl rollout to rollback. But if you do that, you will end up running old code on new/updated configmaps (bad!). We already have a Right Way to rollback (sync to earlier revision of terra-helmfile), so let’s not keep old revisions around and potentially leave the door open for devs to do the Wrong Thing. (Plus, old deployments clutter up the ArgoCD UI)
+* Standard labels are added to pod template metadata, not just deployment metadata as defined in K8s [recommended lables](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/).
```
Comment (choover-broad) [inline actions/terra-k8s-linter/policy/deployments/README.md]: It looks like the policy just checks deployment metadata, so maybe reword this as:

> Standard labels are added to deployment metadata

File: actions/terra-k8s-linter/policy/deployments/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+### Deployment Linting policies
+
+#### General policies
+
+Policies can be found in [deny.rego](https://github.com/DataBiosphere/github-actions/blob/master/actions/terra-k8s-linter/policy/deployments/deny.rego). These are policies that can't don't fall in s particular group and are general in nature.
+* Deployments should set revisionHistoryLimit to 0
+    * The only reason to keep old revisions around is to use kubectl rollout to rollback. But if you do that, you will end up running old code on new/updated configmaps (bad!). We already have a Right Way to rollback (sync to earlier revision of terra-helmfile), so let’s not keep old revisions around and potentially leave the door open for devs to do the Wrong Thing. (Plus, old deployments clutter up the ArgoCD UI)
```
Comment (choover-broad) [inline actions/terra-k8s-linter/policy/deployments/README.md]: Ha! I think I wrote this paragraph originally. I think it can be reworded as:

> This prevents developers from attempting to rollback deployments using `kubectl rollout`, which could lead to an old Deployment running with new ConfigMaps; the correct way to roll back is to sync ArgoCD to an earlier revision of terra-helmfile.

**Review by choover-broad — DISMISSED** [review]
Approved, left some documentation comments in-line

File: actions/terra-k8s-linter/policy/deployments/general_policies.rego (hunk 1 above)
Comment (knapii-developments) [reply actions/terra-k8s-linter/policy/deployments/general_policies.rego]: Not a nit-pick at all. I appreciate you catching this! e85d961

File: actions/terra-k8s-linter/policy/deployments/README.md (hunk 3 above)
Comment (knapii-developments) [reply actions/terra-k8s-linter/policy/deployments/README.md]: a1e3d2c

File: actions/terra-k8s-linter/policy/deployments/README.md (hunk 2 above)
Comment (knapii-developments) [reply actions/terra-k8s-linter/policy/deployments/README.md]: 4cc248d

**Review by jack-r-warren — APPROVED** [review]
